src/Widgets/calendar/Util.vala

```diff
@@ -71,22 +73,25 @@ namespace Util {
                 return new GLib.TimeZone (tzid);
             }
         }
-
-        //FIXME See https://github.com/libical/libical/pull/513
         // If tzid fails, try ICal.Time.get_timezone ()
-        ICal.Timezone* timezone = date.get_timezone ();
+        unowned ICal.Timezone? timezone = null;
+        if (timezone == null && date.get_timezone () != null) {
```

Comment:
It seems unnecessary to test for `null` here as you have just set `timezone` to `null` - or am I missing something?

Reply:
You're right, applied the same solution as in https://github.com/elementary/calendar/pull/722